### ISSUE TITLE
question about label mistake

### ISSUE THREAD

**Issue by yuyijie1995** [experiencor/keras-yolo3#74]
![1 9](https://user-images.githubusercontent.com/30818917/40541355-fcf33790-604d-11e8-931d-20d6ab083eb1.jpg)
There is a mistake about label,I already check the xml and config.json ,but I am still confused.
![image](https://user-images.githubusercontent.com/30818917/40541484-8131ca26-604e-11e8-81db-ab27fbfbdd1d.png)
![image](https://user-images.githubusercontent.com/30818917/40541494-8b0b144e-604e-11e8-8487-b49f00a62008.png)
I hope that you can help me check it.Thank you very much

**Comment by yuyijie1995** [comment]
setting one label is OK,once I used more than one label something was wrong 
![image](https://user-images.githubusercontent.com/30818917/40571593-401b17f6-60ce-11e8-8aba-dd4e66fdcc72.png)

**Comment by yuyijie1995** [comment]
@mohamed29ac

**Comment by illw84u** [comment]
@yuyijie1995 did you solve the problem? I got similar issue here.

**Comment by yuyijie1995** [comment]
@illw84u  YES,I realized that when the code start to train ,it will choose label arrange randomly,such as my original label arrange  [person,bycicle,car] will become [bycycle,car,person],I need to change some code about it ,or you can change the label arrange in your config.json when the trainning step finish to predict the test image.

**Comment by jasondoule** [comment]
@yuyijie1995 @illw84u Got similar issue here. Can you elaborate more in how to resolve this? thanks!

**Comment by yuyijie1995** [comment]
@jasondoule alphabately write the label

**Comment by jasondoule** [comment]
@yuyijie1995 I see - thanks!
